libraries/hts221_driver/hts221.c

```diff
@@ -0,0 +1,301 @@
+
+
+#include "hts221.h"
+#include "string.h"
+#include <stdio.h>
+#include <stdint.h>
+
+#define DEBUG_LOG_MODULE_NAME "HTC221_DRIVER"
+#define DEBUG_LOG_MAX_LEVEL LVL_DEBUG
+#include "debug_log.h"
+
+/** Maximum I2C write transfer */
+#define MAX_WRITE_SIZE 16
+
+/** Maximum I2C read transfer */
+#define MAX_READ_SIZE 16
+
+#define HTS221_ADDR 0x5F
+
+#define AV_CONF_RESERVED_BITS (0xC0)   ///< Reserved bits in the AV_CONF register.
+#define CTRL_REG1_RESERVED_BITS (0x78) ///< Reserved bits in the CTRL_REG1 register.
+#define CTRL_REG2_RESERVED_BITS (0x7C) ///< Reserved bits in the CTRL_REG2 register.
+#define CTRL_REG3_RESERVED_BITS (0x3B) ///< Reserved bits in the CTRL_REG3 register.
+
+static i2c_conf_t m_i2c_conf = {
+    .clock = 100000,
+    .pullup = false,
+};
+
+/**@brief Configuration struct.
+ */
+typedef struct
+{
+    hts221_evt_handler_t evt_handler; ///< Event handler, called when data is ready.
+    bool enabled;                     ///< Driver enabled flag.
+    hts221_calib_t calib;             ///< Calibration struct.
+} htc221_t;
```

Comment:
I see both htc211 and hts211 in this file. Is that intended?

Reply:
No it is not intended.
Will be corrected